src/Init/Control/Except.lean

```diff
@@ -13,6 +13,11 @@ import Init.Coe
 namespace Except
 variable {ε : Type u}
 
+/-- Returns true if the value is `Except.ok`.-/
+def isOk : Except ε α → Bool
```

Comment:
This is `Except.toBool`, but we should still have this alias I think (you can deduplicate the implementation though).

Reply:
ah yes oops